bitnami/harbor/templates/_helpers.tpl

```diff
@@ -230,6 +230,9 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
   {{- if and (eq .Values.redis.enabled false) .Values.externalRedis.password -}}
     {{- .Values.externalRedis.password -}}
   {{- end -}}
+  {{- if and (eq .Values.redis.enabled true) .Values.redis.password -}}
```

Comment:
I don't like using `eq` function to compare with booleans:
```
-   {{- if and (eq .Values.redis.enabled true) .Values.redis.password -}}
+  {{- if and .Values.redis.enabled .Values.redis.password -}}
```
It also applies to line 230:
```diff
-  {{- if and (eq .Values.redis.enabled false) .Values.externalRedis.password -}}
+  {{- if and (not .Values.redis.enabled) .Values.externalRedis.password -}}
```